Gemfile

```diff
@@ -8,6 +8,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'jquery-rails'
 gem 'devise'
+gem 'font-awesome-rails', require: 'font-awesome-rails'
```

Comment:
It was part of the install docs that we require it somewhere. Seemed like here was the best place.